homeassistant/components/reolink/host.py

```diff
@@ -138,6 +140,32 @@ async def async_init(self) -> None:
 
         await self.subscribe()
 
+        _LOGGER.debug(
+            "Waiting for initial ONVIF state on webhook '%s'", self._webhook_url
+        )
+        try:
+            await asyncio.wait_for(self._webhook_reachable.wait(), timeout=15)
```

Comment:
We prefer using `async_timeout` instead of `asyncio.wait_for`. The former has less overhead. Search for examples in the project.